admin-dev/themes/new-theme/js/pages/order/view/order-view-page.js

```diff
@@ -337,4 +317,42 @@ export default class OrderViewPage {
   getActivePage() {
     return $(OrderViewPageMap.productsTablePagination).find('.active span').get(0);
   }
+
+  refreshProductsList(orderId) {
+    const $loadingSpinner = $(OrderViewPageMap.productsPanel).find('.spinner-order-products-container#orderProductsLoading');
+    $loadingSpinner.show();
+
+    const $tablePagination = $(OrderViewPageMap.productsTablePagination);
+    const numRowsPerPage = $tablePagination.data('numPerPage');
+    const initialNumProducts = parseInt($(OrderViewPageMap.productsCount).html(), 10);
```

Comment:
```suggestion
    const initialNumProducts = $(OrderViewPageMap.productsTableRows).length;
```
I didn't make this change last time I modified this, but it's probably more stable this way 😉